scripts/sched_ftrace.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+duration=${1:-5}
+out=${2:-"$(pwd)/sched.ftrace"}
+cd /sys/kernel/debug/tracing
+echo 1 > tracing_on
+echo 1 > events/sched/sched_switch/enable
+timeout "${duration}" cat trace > "${out}.tmp"
```

Comment:
Should we read trace_pipe maybe?

Reply:
I also think it should be trace_pipe (not trace).

Reply:
That seems to work, will update.